testdata/serviceconfig.yml

```diff
@@ -30,7 +30,17 @@ rules:
   - kind: access-logs
     params:
       logName: "access_log"
-      logFormat: 0 # Common Log Format
+      log:
+        log_format: 1 # apache common log format
+        template_expressions:
```

Comment:
Can we add a (duplicate) set of labels here to match the template_exprs (for completeness sake)?